iOS Client/DevotionsController/MBDevotionsViewController.swift

```diff
@@ -12,11 +12,15 @@ import ReSwift
 class MBDevotionsViewController: UIViewController, StoreSubscriber, UITableViewDelegate, UITableViewDataSource {
 
     @IBOutlet weak var tableView: UITableView!
-    var devotions: [MBDevotion] = []
+    var devotions: [LoadedDevotion] = []
+    var cellReusableId: String = "DevotionTableViewCell"
     override func viewDidLoad() {
         super.viewDidLoad()
         tableView.delegate = self
         tableView.dataSource = self
+        tableView.rowHeight = UITableViewAutomaticDimension
+        
+        tableView.register(UINib(nibName: cellReusableId, bundle: nil), forCellReuseIdentifier: cellReusableId)
```

Comment:
it's kind of a coincidence that the nibName is the same as the cellReusableId. I'd still hard-code the nibName so the reuse ID value can change without hecking up this code

Reply:
`nibName` should be hard-coded in my opinion. The nibName and the cellReuseIdentifier should be independent of one another